README.md

```diff
@@ -38,6 +38,8 @@ Change summary
 | Spitfire   | 35 -> 25 (3 -> 4 BTK) | 25 -> 20 (4 -> 5 BTK)  | 20 -> 17 (5 -> 6 BTK)  |
 | EVA-8      | 200 -> 150            | 10 (_vanilla_)         | -                      |
 | Melee      | 100 -> 70             | -                      | -                      |
+| Re45       | 20 -> 15 (4 -> 7 BTK) | 13 -> 12 (7 -> 9  BTK) | 9  -> 8 (12 -> 13 BTK) |
```

Comment:
I think we should start with +2 BTK instead of +3, or..?
Also RE-45 is the correct name